### PR TITLE
Client Telemetry: Fixes Region contacted information in case of multiple values

### PR DESCRIPTION
**Pull request by sourabh1007**
## Description

Right now,if "RegionContacted" has multiple region information, in that it is getting region URIs but if there is single region then getting region name. So making it consistent i.e Region name.

image
## Type of change
- [] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: Microsoft.Azure.Cosmos/src/Telemetry/ClientTelemetryHelper.cs [hunk 1]
```diff
@@ -185,15 +185,15 @@ internal static string GetContactedRegions(CosmosDiagnostics cosmosDiagnostics)
             }
 
             StringBuilder regionsContacted = new StringBuilder();
-            foreach ((_, Uri uri) in regionList)
```
Comment (ealsur) [inline Microsoft.Azure.Cosmos/src/Telemetry/ClientTelemetryHelper.cs:188]: Any tests need to be added/updated?

Comment (sourabh1007) [reply Microsoft.Azure.Cosmos/src/Telemetry/ClientTelemetryHelper.cs:188]: added

**Review by ealsur — APPROVED** [review]

**Review by j82w — APPROVED** [review]

File: Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.Performance.Tests/Contracts/BenchmarkResults.json [hunk 2]
```diff
@@ -21,20 +21,20 @@
   "MockedItemBenchmark.QuerySinglePage;[Type=OfTWithClientTelemetryEnabled]": 2244658.0,
   "MockedItemBenchmark.QuerySinglePage;[Type=OfTWithDiagnosticsToString]": 2310842.0,
   "MockedItemBenchmark.QuerySinglePage;[Type=Stream]": 1005038.0,
-  "MockedItemBenchmark.ReadFeed;[Type=OfT]": 560692.0,
+  "MockedItemBenchmark.ReadFeed;[Type=OfT]": 38680.0,
```
Comment (j82w) [inline Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.Performance.Tests/Contracts/BenchmarkResults.json:24]: Please revert these changes. Nothing in this PR should have cause these values to change so much.

**Review by j82w — APPROVED** [review]
